src/Epic/index.tsx

```diff
@@ -171,11 +179,13 @@ export const Epic: React.FC<EpicProps> = (props: EpicProps) => {
                         />
                         {reminderStage && (
                             <ReminderCtaButton
+                                reminderComponent="EPIC"
                                 reminderStage={reminderStage}
                                 reminderOption={reminderOption}
                                 ophanComponentId={ophanComponentId as string}
                                 trackClick={trackClick}
                                 fetchEmail={fetchEmail}
+                                userStyles={defaultColors}
```

Comment:
What about calling this prop `colors` or similar? I think `userStyles` isn't always accurate (in this case they're hardcoded for example).

Reply:
I'll take this forward in the next PR